common/lib/xmodule/xmodule/tests/test_sequence.py

```diff
@@ -111,6 +111,11 @@ def test_student_view_first_child(self):
     def test_student_view_last_child(self):
         html = self._get_rendered_student_view(self.sequence_3_1, requested_child='last')
         self._assert_view_at_position(html, expected_position=3)
+        html = self._get_rendered_student_view(
+            self.sequence_3_1,
+            extra_context=dict(next_url='NextSequential', prev_url='PrevSequential', efischer_silly_var=True),
+        )
+        self._assert_view_at_position(html, expected_position=1)
```

Comment:
These lines were copied from `test_render_student_view` above, and were used to illustrate what I mean about the test ordering. Without the above `efischer_silly_var` variable, the code from `test_render_student_view` test will *always* fail, because the code in `test_student_view_last_child` (specifically, `self._get_rendered_student_view(..., requested_child='last')`) sets a value on the sequence module that impacts later tests.